exercises/randomgenerator.js

```diff
@@ -2,21 +2,34 @@
 
 class RandomGenerator{
 	constructor(min, max){ 
```

Comment:
The “class constructor” of an object is the section which assigns "values" to different "keys" based on what is passed in when the object is instantiated. For example, in this prototype, when a new "randomgenerator" object is created the “min” and “max” values are passed in and assigned to the object that is created. One could imagine that if you we making the constructor for a "student" object that it would contain such key/value pairs as "name", "age", "grade", and maybe even "attendance". If this was the case, when the new object was created and values were passed in for these "keys" they would be assigned to the new object. Great work on this!